introductions/getting-started.md

```diff
@@ -1,38 +1,27 @@
 # Getting Started
 
-## 1. Make Sure You Have a Github Account
-The first step is to make sure you have a Github Account.
-
-Here is the link if you do have one already:
-- [Github account creation link](https://github.com/join)
-
-## 2. Log into Pangeo Binder
-Next, sign in and authenticate the Pangeo Binder, which is the platform we will use for the workshop:
-- [Pangeo Binder Link](https://aws-uswest2-binder.pangeo.io)
+## About
 
-## 3. Launch our Environment
-Now that we have our authentication set up, we can access our content!
+The repository was created from [a ProjectPythia cookbook template](https://github.com/ProjectPythiaCookbooks/cookbook-template).
+This template brings with it all machinery to enable full featured GitHub workflows, including building a docker image, running and rendering
+Jupyter Notebooks and compiling a website using Sphinx and the [JupyterBook](https://jupyterbook.org/intro.html) theme,
 
-Use the following link to launch into the binder:
-- [Binder Link](https://aws-uswest2-binder.pangeo.io/v2/gh/openradar/erad2022/main?labpath=notebooks)
+## Customizing
 
-If you are having issues with that (ex. it is taking a long time), try using the following link:
-```
-https://hub.aws-uswest2-binder.pangeo.io/user/{your_github_username}/lab
-```
-Where you replace `{your_github_username}` with your github username (ex. `mgrover1`)
+If there is a package missing for your use-case you can just add the package to [binder/environment.yml](https://github.com/openradar/erad2022/blob/main/binder/environment.yml) and activate a new build (see below).
+Notebooks might be added to the [notebooks-folder](https://github.com/openradar/erad2022/tree/main/notebooks).
 
-## Description of the build workflow
+## Build workflow
```

Comment:
Perhaps adding a "general philosophy" here would be helpful. For example,

"We work on pre-building the environment and toolkit so you can deploy this on the platform of your choice! Making the Open Radar Science stack available across different environments is essential to ensure reproducibility. We take advantage of ["containerization"](https://www.ibm.com/cloud/learn/containerization), which enables packaging all of these tools together so you can deploy it on the platform of your choice. In this case, we are deploying a software stack which includes a variety of different languages (ex. Python, C++, C) and deploy it on a cloud platform. We use [Docker](https://www.docker.com) as our main containerization tool."

Feel free to suggest other overarching ideas,